doc/manual/manual.tex

```diff
@@ -1568,7 +1568,41 @@ \section{Installation}
 This is a brief explanation of how to install all the required software and
 \aspect{} itself.
 
-\subsection{Prerequisites}
+\subsection{System prerequisites}
+
+In order to install \aspect{} and its dependencies, you should
+have your system set up to be able to compile and link programs. Additionally,
+\aspect{} needs a number of widely used libraries that are available
+for most operating systems. Therefore, you will need compilers for C, C++ and
+Fortran, the GNU make system, the CMake build system, development versions of
+the BLAS and LAPACK libraries and the zlib library, which is used for
+compressing the output data. To use more than one process for your computations
+you will need to install a development version of a MPI library and the
+necessary executables to run MPI programs. There are some optional packages
+for additional features, like the HDF5 libraries for additional output formats
+and PETSC for alternative solvers, but these are not strictly required.
+Numdiff is used for checking \aspect{}'s test results with reasonable accuracy.
```

Comment:
Might be worth pointing out that PETSc and numdiff may not be available through the package manager of all systems. (I suppose so, at least, but can't really say.)